src/Css.elm

```diff
@@ -3978,6 +3987,110 @@ backgroundPosition4 (Value horiz) (Value horizAmount) (Value vert) (Value vertAm
 
 
 
+-- BACKGROUND REPEAT --
+
+
+{-| Sets [`background-repeat`](https://css-tricks.com/almanac/properties/b/background-repeat/)
+
+    backgroundRepeat repeat
+
+    backgroundRepeat repeatX
+
+If you need to set horizontal and vertical direction separately, see
+[`backgroundRepeat2`](#backgroundRepeat2)
+
+-}
+backgroundRepeat :
+    Value
+        { repeat : Supported
+        , repeatX : Supported
+        , repeatY : Supported
+        , space : Supported
+        , round : Supported
+        , initial : Supported
+        , unset : Supported
+        , inherit : Supported
+        }
+    -> Style
+backgroundRepeat (Value repeat) =
+    AppendProperty ("background-repeat:" ++ repeat)
+
+
+{-| Sets [`background-repeat`](https://css-tricks.com/almanac/properties/b/background-repeat/) along the horizontal axis, then the vertical axis.
+
+    backgroundRepeat2 repeat space
+
+    backgroundRepeat2 space round
+
+If you only need to set one value for both, see
+[`backgroundRepeat`](#backgroundRepeat) instead.
+
+-}
+backgroundRepeat2 :
+    Value
+        { repeat : Supported
+        , space : Supported
+        , round : Supported
+        , initial : Supported
+        , unset : Supported
+        , inherit : Supported
+        }
+    ->
+        Value
+            { repeat : Supported
+            , space : Supported
+            , round : Supported
+            , initial : Supported
+            , unset : Supported
+            , inherit : Supported
+            }
+    -> Style
+backgroundRepeat2 (Value horiz) (Value vert) =
+    AppendProperty ("background-repeat:" ++ horiz ++ " " ++ vert)
+
+
+{-| [Repeat a background](#backgroundRepeat).
+-}
+repeat : Value { provides | repeat : Supported }
+repeat =
+    Value "repeat"
+
+
+{-| [Repeat a background](#backgroundRepeat) side to side.
+-}
```

Comment:
On these `Value` docs, can we get links to https://developer.mozilla.org/en-US/docs/Web/CSS/background-repeat#Values and mention that this compiles to the `"repeat-x"` value in CSS? (Similar to [this](https://github.com/rtfeldman/elm-css/pull/408/files#diff-854293bbb03f363ae4c16745b0af2a75R1200), for example.)

Code examples would also be great, even if they're copy/pasted from the `background` docs. 😄